WordPress/src/main/java/org/wordpress/android/ui/sitecreation/previews/SitePreviewViewModel.kt

```diff
@@ -150,7 +150,7 @@ class SitePreviewViewModel @Inject constructor(
         _onOkButtonClicked.value = createSiteState
     }
 
-    private fun showFullscreenErrorWithDelay() {
```

Comment:
We might want to consider keeping this method private. I think the need to call this method from the view layer is an indication that we have a business logic in the view layer. I believe the view layer should just inform the VM about the events and not make any business logic decisions. We might just want to call "viewModel.onError/onPreviewLoadingFailed" from the Fragment. Wdyt? 

SideNote: If you decide to implement this, it'd also make sense to write a simple unit test for this scenario.

Reply:
Updated to  private function with the changes made to support the preview empty screen.